homeassistant/components/group/sensor.py

```diff
@@ -318,7 +318,8 @@ def __init__(
             self._attr_name = f"{DEFAULT_NAME} {sensor_type}".capitalize()
         self._attr_extra_state_attributes = {ATTR_ENTITY_ID: entity_ids}
         self._attr_unique_id = unique_id
-        self.mode = all if mode is False else any
+        self._ignore_non_numeric = ignore_non_numeric
+        self._mode = all if ignore_non_numeric is False else any
```

Comment:
```suggestion
        self.mode = all if ignore_non_numeric is False else any
```

Reply:
If I may, what's the attachment to `mode`? The only tangentially related references I can find are in the [legacy](https://www.home-assistant.io/integrations/group/#old-style-groups) `Group` class which I believe to be whole independent from `SensorGroup` and it's parent `GroupEntity` classes.

Reply:
From outside of sensor group nothing I believe more than this is much based from the `min_max` integration. 

I just prefer to leave it as it was as I don't see why this should change.

Reply:
Ok